README.md

```diff
@@ -314,6 +316,51 @@ Name            | Effect |
 `vagrant-box`    | Build the Vagrant box image using `packer` |
 `vagrant-prep`    | Shortcut for building everything needed for `make run` |
 
+# Deployment Customizations
+
+## Customize The Application Domain
+
+In a standard installation the domain used by applications pushed to
+CF is the same as the domain configured for CF itself.
+
+This document describes how to change this so that CF and applications
+use separate domains.
+
+:warning: The changes described below will work only for CAP 1.3.1,
+and higher.  They do not work for CAP 1.3, and will not work for the
```

Comment:
[nit] There is an extra space between the 2 sentences that shouldn't exist.

Reply:
I have seen both one and two spaces in other docs, both some to be ok.
... ok, go one space. likely more consistent with rest of file.

Reply:
Yeah. If the entire file was using 2 spaces I could agree to just keep 2 spaces forever here. (or I would advocate for 1 space). :)

Reply:
I think it makes perfect sense to have 2 spaces for text that is typically displayed in monospaced fonts. Once converted to HTML the double space is optimized away anyways.

So from a usability point of view I actually prefer 2 spaces after each sentence in Markdown documents and source code comments. 😛

Reply:
Done with commit 05aaa3c57e9da760304a8f1cd5e48362695b77d0